.travis.yml

```diff
@@ -122,6 +122,7 @@ jobs:
       env: GO_TEST_TIMEOUT=20m
       script:
         - mysql -u root -e 'SHOW VARIABLES LIKE "%version%";'
+        - go test ./storage/mysql/...
```

Comment:
Eh? Is it not run?

Reply:
It seems like only the "coverage" target runs the unit-tests. It makes sense though to run the specific ones here.

Reply:
The tests are run in other targets, but it makes sense to run the DB-specific tests again with different DB implementations installed.